baseapp/msg_service_router.go

```diff
@@ -39,27 +39,32 @@ func (msr *MsgServiceRouter) Handler(methodName string) MsgServiceHandler {
 
 // RegisterService implements the gRPC Server.RegisterService method. sd is a gRPC
 // service description, handler is an object which implements that gRPC service.
+//
+// This function PANICs if it is called before the the service `Msg`s have been
+// registered using RegisterInterfaces.
```

Comment:
I believe "running `RegisterInterfaces` before running `RegisterService`" is also worth mentioning in docs.